test/libsolidity/SolidityExecutionFramework.cpp

```diff
@@ -86,5 +87,9 @@ bytes SolidityExecutionFramework::compileContract(
 	else
 		obj = m_compiler.object(_contractName.empty() ? m_compiler.lastContractName() : _contractName);
 	BOOST_REQUIRE(obj.linkReferences.empty());
+	if (m_showMetadata)
+		cout << "metadata: "
```

Comment:
The `<<` should go to the end of the lines.

Reply:
what do you mean?

Reply:
ahh :)